.github/workflows/ci.yml

```
@@ -1,34 +1,55 @@
+
+      - name: 🚀 Criar Release no GitHub
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ github.ref_name }}
+          name: Release ${{ github.ref_name }}
+          body: ${{ env.BODY }}
+          draft: false
+          prerelease: false
         env:
-          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}  # Ou remova essa linha se estiver usando GITHUB_TOKEN padrão
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)